### PR TITLE
multiclass sample data and expected results

### LINKED ISSUE #85: Add multiple classifications to sample data, and validations
The sample review dataset should include 1-3 classifications for each sample. We also need the expected classifications to validate against, like in sample_reviews_data_expected_classes.csv. This can be done by selecting e.g. 5 of the samples as training, and then fill in what we expect for the other 30 samples' classifications. Then we can test the new experiment runner experiment.py on the data, and validate the 30 classifications are identical to what we expect.

### PR DESCRIPTION
**Pull request by akhilaananthram**
The data has between 1-3 classes.  The expected results are what you would get if you trained a Random SDR model on the first 5 inputs.  This uses k=1 for the KNN classifier.

There is also a file for the expected results for the fingerprint classifier using the raw overlap as the distance metric for KNN

Fixes #85 

@BoltzmannBrain

### CODE REVIEW

**Comment by BoltzmannBrain** [comment]
@akhilaananthram looks good so far. These expected classifications are specific to `ClassifcationModelRandomSDR` yes? So please add CSVs of expected classifications for `ClassificationModelFingerprint` and `ClassificationModelEndpoint`.

**Comment by akhilaananthram** [comment]
@BoltzmannBrain: added expected classifications for fingerprint and endpoint

File: data/sample_reviews_multi/sample_reviews_data_random_expected_classes.csv [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+5,,"blah blah like anyone will read this anyway",not helpful,,
+6,,"sugestions vote box could be used for i wednesday lunch ii office improvements iii game movie night requests etc",kitchen,not helpful,
+7,,"better snacks stop people from stealing my food even after i labeled it its not right and it makes me ngry",kitchen,not helpful,
+8,,"stay golden pony boy",,,
+9,,"once a week show at lunch i e lecture tv show documentary etc",kitchen,not helpful,
+10,,"more video games board games and scavenger hunts",work/life balance,,
+11,,"love teh office i wouldn t change a thing",not helpful,,
+12,,"better more healthy snacks otherwise the office is perfect",kitchen,,
+13,,"i can t think of a better workplace environment",not helpful,kitchen,work/life balance
+14,,"identifier deleted gives each employee $ to decorate their workspace just sayin",,,
```
Comment (BoltzmannBrain) [inline data/sample_reviews_multi/sample_reviews_data_random_expected_classes.csv]: What happened to `$10,000`?

Comment (akhilaananthram) [inline data/sample_reviews_multi/sample_reviews_data_random_expected_classes.csv]: This is the output of the text preprocessor.  the basic one removes numbers.

**Comment by BoltzmannBrain** [comment]
@akhilaananthram a few things...
- It looks like many of the samples changed from the originals... why? Maybe from reading in the csv and then writing it out there was some loss of characters. Regardless they must be identical to the samples in sample_reviews_data_training.csv.
- Why did you change some of the samples?
- Training on only five samples, particularly the first five, doesn't tell us much about the resulting classifications. Better would be training on samples 26-35; most (but not all) classes trained for, and the test set has all the tricky nuances.

File: data/sample_reviews_multi/sample_reviews_data_random_expected_classes.csv (hunk 1 above)
Comment (BoltzmannBrain) [inline data/sample_reviews_multi/sample_reviews_data_random_expected_classes.csv]: okay

**Comment by BoltzmannBrain** [comment]
On second thought, it will be okay to leave the preprocessed samples in the expected classes CSVs. But what text-preprocessing does this include?

**Comment by akhilaananthram** [comment]
This just includes the tokenizing which removes all characters that aren't in a-z or $.  It also makes it lowercase.  I left the tokenized version in the expected results so we would know what text preprocessing was done to achieve those results.

**Comment by BoltzmannBrain** [comment]
@akhilaananthram I'm a little confused by the expected classifications for the first five samples... shouldn't they either be exactly correct, or not even filled in (because we only train on them, not test)?

**Comment by akhilaananthram** [comment]
I wrote out what would happen if we tested on them after.  I'm unclear on why they would have to be exactly correct.  If we are always returning 3 classes and one of the training examples has less than 3 classes, wouldn't we return a label that isn't correct?

**Comment by BoltzmannBrain** [comment]
:+1:

**Pull request merged**